### DISCUSSION TITLE
Yubikey - Require hardware key to save

### DISCUSSION THREAD

**Discussion by InAUGral** [keepassxreboot/keepassxc#6192] [Q&A] [answered]
Hi all,

I have recently decided to make use of my Yubikey to open/unlock my database with a password + yubikey challenge. This method works fine the past few days although I have noticed that if I make any changes then I must also insert the yubikey for it to save. I would like to turn this feature off if possible as I do not need its necessary in my case.

**Comment by droidmonkey** [comment, marked as answer]
You need to keep it plugged in to save because we issue a new challenge on each save. This prevents replay attacks and increases security. Making this feature optional has been proposed on the issue board already. It's under consideration.

**Reply by InAUGral** [reply]
Thanks for the reply. It would be good to have the option. Ideally I would like to disable this on the desktop but still keep Yubikey authentication when saving from Mobile if that would at all be possible to have best of both worlds.
